apps/web-connect/src/features/checkout/Checkout.tsx

```diff
@@ -40,10 +42,16 @@ export function Checkout() {
     }, [prefilledPromoCode, setPromoCode]);
 
     function returnToClient() {
-        const hash = mintWithEth.data?.hash ?? mintWithXai.data?.hash;
-        window.location = `xai-sentry://purchase-successful?txHash=${hash}` as unknown as Location;
+        resetTransactions();
```

Comment:
Was tehre a difference when coming from the desktop client so the app would return to the client after minting ?

Reply:
This has been added back in